src/quickstart/install-cli-tools.md

```diff
@@ -10,13 +10,20 @@ Install the latest `deis` client for Linux or Mac OS X with:
 The installer places the `deis` binary in your current directory, but you
 should move it somewhere in your $PATH:
 
+    $ sudo ln -fs $PWD/deis /usr/local/bin/deis
+
+*or*:
+
     $ sudo mv $PWD/deis /usr/local/bin/deis
 
 Check your work by running `deis version`:
 
     $ deis version
     2.1.0
```

Comment:
I see we should update this to `v2.7.0` (not necessarily in this PR). It must have gotten missed because it didn't prepend the `v`. 😞